src/ng/q.js

```diff
@@ -504,6 +504,21 @@ function qFactory(nextTick, exceptionHandler) {
    * Combines multiple promises into a single promise that is resolved when all of the input
    * promises are resolved.
    *
+   * ```js
+   *   promiseC = $q.all([promiseA, promiseB]).then(function(result) {
+   *     // success: do something and resolve promiseC
```

Comment:
Since you are at it, it might be useful to somehow indicate what `result` looks like (i.e. an array with the resolved values for `promiseA`, `promiseB`).